src/prefect/core/flow.py

```diff
@@ -159,6 +160,12 @@ def __init__(
         self.storage = storage
         self.result_handler = result_handler
 
+        if result_handler:
+            warnings.warn(
+                "DEPRECATED: the result_handler Flow option is deprecated, please provide a 'prefect.engine.result.Result' object to the 'result' option instead.",
```

Comment:
```suggestion
                "DEPRECATED: the result_handler Flow option will be deprecated in 0.11.0, and removed in 0.12.0, in favor of the `result` option instead.",
```

@joshmeek @wagoodman Here's my take on the copy. Then I think for 0.11.0 we can add in the original second clause to "please provide...." since according to our plans it will be the recommended arg by then.

Reply:
I like it 👍